train_model.py

```diff
@@ -202,11 +203,13 @@ def main():
     )
     parser.add_argument(
         "--var_leads_metrics_watch",
-        type=dict,
-        default={},
-        help="Dict with variables and lead times to log watched metrics for",
+        type=str,
+        default="{}",
+        help="JSON string with variables and lead times to log watched metrics"
+        # e.g. '{"var1": [1, 2], "var2": [3, 4]}'
```

Comment:
Can we move the example comment into the help message? Would be useful to see when running `python train_model.py --help`

Reply:
I can fix this myself

Reply:
Actually nvm. I can't get this argument to work, so you'll have to take a look at this @sadamov. It used to be the case that this was a dict mapping from variable index to a list of lead times, rather than variable names. Was this changed to names now?

This is used here, where the dict keys index `var_nums`. I guess this should be the var names?
https://github.com/mllam/neural-lam/blob/879cfec1b49d0255ed963f44b3a9f55d42c9920a/neural_lam/models/ar_model.py#L474-L476

Reply:
![image](https://github.com/mllam/neural-lam/assets/45732287/266d825f-fab5-4a69-b816-07622f82df5e)